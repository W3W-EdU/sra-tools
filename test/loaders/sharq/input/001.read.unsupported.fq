@SRA56789 200/1
GTCGCTTCTCGGAAGNGTGAAAGACAANAATNTTNN
+BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540
40 3 1 22 17 18 34 8 13 21 -3 7 5 0 -0 -5 1 -0 7 3 2 -3 -3 3 -1 -1 4 -5 5 -2 -2 -5 -0 -1 -5 -5
@200/2
GAGCACGAGGACCCANNNATCAACGACNANNNGANN
+666_BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:118:250
40 40 40 40 40 40 40 22 33 40 16 29 4 -1 1 -5 -5 -5 7 -3 1 -1 -1 6 -1 14 -1 -5 -2 -5 -5 -5 6 -1 -5 -5
@200/3
ACTGGAGCACGANAGNCCCNCNACTNANANCNGNNN
+BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:212:211
40 40 40 40 40 27 40 40 12 8 5 6 -5 -3 0 -5 -4 0 4 -5 7 -5 7 1 0 -5 -0 -5 -1 -5 -0 -5 7 -5 -5 -5
