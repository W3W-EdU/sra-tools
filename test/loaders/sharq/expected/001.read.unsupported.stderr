[error] [code:100] Defline not recognized [001.read.unsupported.fq:1]
